cmd/changelog/main.go

```diff
@@ -0,0 +1,351 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"reflect"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var (
+	mergeRequestRegexp = regexp.MustCompile(`Merge-pull-request-([\d]+)`)
+	prefixRegexp       = regexp.MustCompile(`^.+: (.+)`)
+	releaseRegexp      = regexp.MustCompile(`(release-\d\.\d)`)
+	latestHashRegexp   = regexp.MustCompile(`<!--Latest hash: (.+)-->`)
+
+	version_sectionRegExp     = regexp.MustCompile(`^(\d.\d)`)
+	backports_sectionRegExp   = regexp.MustCompile(`### Backports\n((.+\n)+)`)
+	enhancement_sectionRegExp = regexp.MustCompile(`### Enhancements\n((.+\n)+)`)
+	bugfix_sectionRegExp      = regexp.MustCompile(`### Bug fixes\n((.+\n)+)`)
+	other_sectionRegExp       = regexp.MustCompile(`### Others\n((.+\n)+)`)
+	misc_sectionRegExp        = regexp.MustCompile(`### Misc\n((.+\n)+)`)
+
+	// PR categories
+	categories = map[string]*regexp.Regexp{
+		"BugFix":      regexp.MustCompile(`- \[[xX]\] Bugfix`),
+		"Enhancement": regexp.MustCompile(`- \[[xX]\] Enhancement`),
+		"Other":       regexp.MustCompile(`- \[[xX]\] Others`),
+		"Backporting": regexp.MustCompile(`- \[[xX]\] Backporting`),
```

Comment:
It looks like these strings repeat more times in various forms. What about to unify it (I guess in the PR template as well) and make them as constants here?